<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聊天室客户端</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleLabel {
    background: white;
    border: none;
    border-bottom: 1px solid black;
    border-top: 1px solid black;
    font: 24pt &quot;黑体&quot;;
    color: rgba(255, 255, 255, 1);
    background-color: rgba(0, 170, 127, 255);
}

#mainFrame {
    background: white;
    border: none;
}

#loginFrame {
    background: linear-gradient(to top, rgba(0, 170, 127, 0.2), rgba(0, 170, 127, 0.2));
    border: 1px solid gray;
    padding: 10px;
    border-radius: 25px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">#titleLabel {
    background: white;
    border: none;
    border-bottom: 1px solid black;
    border-top: 1px solid black;
    font: 24pt &quot;黑体&quot;;
    color: rgba(255, 255, 255, 1);
    background-color: rgba(0, 170, 127, 255);
}

#mainFrame {
    background: white;
    border: none;
}

#loginFrame {
    background: linear-gradient(to top, rgba(0, 170, 127, 0.2), rgba(0, 170, 127, 0.2));
    border: 1px solid gray;
    padding: 10px;
    border-radius: 25px;
}</string>
      </property>
      <property name="text">
       <string>陈伟锋的聊天室</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="chatPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTextEdit" name="roomtextEdit"/>
          </item>
          <item>
           <widget class="QListWidget" name="userlistWidget">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="saylineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="sayButton">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="layoutButton">
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="loginPage">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>123</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>175</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QFrame" name="loginFrame">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(243, 230, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QFormLayout" name="mainFrame">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>服务器地址：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="serverEdit">
               <property name="text">
                <string>127.0.0.1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>昵称：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="usernameEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="loginButton">
             <property name="text">
              <string>登录聊天室</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>174</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>123</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
